code/espurna/light.ino

```diff
@@ -474,8 +474,16 @@ void _lightProviderUpdate() {
 // -----------------------------------------------------------------------------
 
 void _lightColorSave() {
+
+    //TODO: Remove this once this code was in a stable release
+    // This force set ch3 to 0 which could be not zero when you update from a old version
+    if (_light_use_white) {
+      setSetting("ch", 3, 0);
```

Comment:
@xoseperez I'm not sure about this. We need to reset this setting when a user upgrade from a old to a new version.

Should I leave it like this? Or do you have a script that run once a user flash/upgrade? Or does it wipe these settings anyway when you flash it?

Reply:
I've rewritten most of the code so this is no longer needed ;)